package.json

```
@@ -31,5 +31,9 @@
     "prettier-plugin-tailwindcss": "^0.6.6",
     "tailwindcss": "^3.4.1",
     "typescript": "^5"
+  },
+  "lint-staged": {
+    "*.{vue, ts, js}": "eslint --fix",
```

```suggestion
    "*.{ts, js, tsx, jsx}": "eslint --fix",
```

- `.vue`
  - 今回.vueファイルは使う予定がないので、不要かな？と思います！
- `ts, js`
  - `tsx, jsx`にもlintを効かせたいので、こちらの追記をお願いします！🙏